src/pages/LoginPage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { signin } from '../api/api';
+import { Redirect } from 'react-router-dom';
+import LoginForm from '../components/LoginForm';
+
+
+export default class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      token: null,
+    };
+    this.signIn = this.signIn.bind(this)
+  }
+
+
+  async signIn(loginData) {
```

Comment:
If you use arrow function, you dont need bind this function above

```suggestion
  async signIn = (loginData) => {
```

Reply:
The asynchronous method does not work

Reply:
Sorry, my fault signIn = async (loginData) => {